crates/napi/src/next_api/project.rs

```diff
@@ -331,11 +331,12 @@
             .unwrap();
         });
     }
-    let options = options.into();
+    let options: ProjectOptions = options.into();
     let container = turbo_tasks
         .run_once(async move {
-            let project = ProjectContainer::new(options);
+            let project = ProjectContainer::new("next.js".into(), options.dev);
             let project = project.resolve().await?;
```

Comment:
Not really your diff, but would this be equivalent and cleaner?

```suggestion
            let project = ProjectContainer::new("next.js".into(), options.dev).await?;
```